### ISSUE TITLE
Kia-Club.htm

### ISSUE THREAD

**Issue by mikejin01** [mikejin01/Slide-Show-Project#1]
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Kia Club</title>

<style type="text/css">
	.popup{
		background:#FFFFFF;
    	position:fixed;
    	width: 400px;
    	height: 320px;
    	border: groove black;
    	box-shadow: 0 0 300px gray;
    	border-radius: 10px;
    	padding:15px;
		left:50%;
    	top:150px;
		margin-left: -200px;
	}
	.presentWindow{
		position:absolute;
	}
	.hide {
		display: none;
	}
	.button{
		background:#CCCCCC;
		border: groove black;
	}
	body {
		font:"Times New Roman", Times, serif;
		margin: 0; 
		padding: 0;
		text-align: center; 
		color:#000000;
	}
	.container {
		top:50px;
		width: 800px; 
		height: 1350px;
		background:#FFFFFF;
		margin: 0 auto; 
		border: 1px solid #000000;
		text-align: left; 
	}
	.menu {
		background:rgba(255,255,255,0.9);
		width:800px;
		height: 50px;
		position: fixed;
		top:0px;
	}
	.space {
		background:#FFFFFF;
		width:800px;
		height: 50px;
	}
	a:link {
		color:#000000;
		text-decoration: none;
	}
	a:visited {
		text-decoration: none;
		color:#000000;
	}
	a:hover {
		color:#999999;
	}
	a:active {
		text-decoration: none;
		color:#FFFFFF;
	}
	.slider1 {
		width: 800px;
		height: 260px;
		overflow: hidden;
	}
	.slider2 {
		width: 800px;
		height: 150px;
		overflow: hidden;
	}
	.slides {
		display: block;
		width: 6000px;
		height: 400px;
		margin: 0;
		padding: 0;
	}
	.slide {
		float: left;
		list-style-type: none;
		width: 800px;
		height: 400px;
	}
	
	/** Carousel Controls **/
	.jcarousel-control-prev,
	.jcarousel-control-next {
    	position:absolute;
    	top: 45%;
    	margin-top: -15px;
    	width: 30px;
    	height: 30px;
    	text-align: center;
    	background: #4E443C;
    	color: #fff;
    	text-decoration: none;
    	text-shadow: 0 0 1px #000;
    	font: 24px/27px Arial, sans-serif;
    	-webkit-border-radius: 30px;
    	-moz-border-radius: 30px;
    	border-radius: 30px;
    	-webkit-box-shadow: 0 0 4px #F0EFE7;
    	-moz-box-shadow: 0 0 4px #F0EFE7;
    	box-shadow: 0 0 4px #F0EFE7;
	}
	
	.jcarousel-control-prev {
    	left: 265px;
	}
	.jcarousel-control-next {
    	right: 265px;
	}

	.roundButton {
    	width: 30px;
    	height: 30px;
    	text-align: center;
    	background: #4E443C;
    	color: #fff;
    	text-decoration: none;
    	text-shadow: 0 0 1px #000;
    	font: 24px/27px Arial, sans-serif;
    	-webkit-border-radius: 30px;
    	-moz-border-radius: 30px;
    	border-radius: 30px;
    	-webkit-box-shadow: 0 0 4px #F0EFE7;
    	-moz-box-shadow: 0 0 4px #F0EFE7;
    	box-shadow: 0 0 4px #F0EFE7;
	}

</style>
</head>

<body>
	
	<div class="container">
    
		<div class="menu" align="center"> 
        <div style="opacity=1.0;">
        <hr />
			<a href="index.html">ABOUT KIA</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="services.htm">VEHICLES</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="contact.htm">SERVICES</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="JS Tutorial/Ajax.html">CONTACT US</a></font></center></p>
		<hr />
        </div> 
        </div>
        
        <div class="space"></div>
        
        <div align="center" class="spaceimg"><img src="images/Kia-Logo.png" width="143" height="100"/></div>
        <div align="center">WE GOT PASSION FOR KIA.</div>
        
        <hr />
        
        <div id="presentWindow" class="presentWindow" >
        <img class="hide" class="hide" id="k900-pic" src="images/Kia-K900-2.jpg" width="800" height="263"/>
        <img class="hide" id="optima-pic" src="images/Kia-Optima-2.jpg" width="800" height="263"/>
        <img class="hide" id="cadenza-pic" src="images/Kia-Cadenza-2.jpg" width="800" height="263"/>
        <img class="hide" id="soul-pic" src="images/Kia-Soul-2.jpg" width="800" height="263"/>
        <img class="hide" id="forte-pic" src="images/Kia-Forte-2.jpg" width="800" height="263"/>
        <img class="hide" id="rio-pic" src="images/Kia-Rio-2.jpg" width="800" height="263"/>
        <img class="hide" id="sorento-pic" src="images/Kia-Sorento-2.jpg" width="800" height="263"/>
        <img class="hide" id="sportage-pic" src="images/Kia-Sportage-2.jpg" width="800" height="263"/>
        </div>
        
        
        
        <div class="slider1" id="slider1" align="center">
    		<ul class="slides">
            	<li class="slide"><img id="headPic" src="images/Kia-Cadenza-2.jpg" width="800" height="263"/></li>
                <li class="slide"><img src="images/Kia-K900-2.jpg" width="800" height="263"/></li>
            	<li class="slide"><img src="images/Kia-Optima-2.jpg" width="800" height="263"/></li>
            	<li class="slide"><img src="images/Kia-Cadenza-2.jpg" width="800" height="263"/></li>
            	<li class="slide"><img src="images/Kia-K900-2.jpg" width="800" height="263"/></li>
        	</ul>
    	</div>  
        <a href="#" class="jcarousel-control-prev" id="previous">&lsaquo;</a>
        <a href="#" class="jcarousel-control-next" id="next">&rsaquo;</a>
        
        <div align="center">
        	<a href="#" data=2 class="roundButton" id="picButton1">1</a>
        	<a href="#" data=3 class="roundButton" id="picButton2">2</a>
        	<a href="#" data=4 class="roundButton" id="picButton3">3</a>
        </div>
        <hr />
        
        <div class="slider2" id="slider2" align="center">
    		<ul class="slides">
            	<li class="slide"><a data-img="k900-pic" id="k900" href="#"><img  src="images/Kia-K900-1.png" width="200" height="150"/></a><a data-img="optima-pic" id="optima" href="#"><img src="images/Kia-Optima-1.png" width="200" height="150"/></a><a data-img="cadenza-pic" id="cadenza" href="#"><img src="images/Kia-Cadenza-1.png" width="200" height="150"/></a><a data-img="soul-pic" id="soul" href="#"><img src="images/Kia-Soul-1.png" width="200" height="150"/></a></li>
            	<li class="slide"><a data-img="forte-pic" id="forte" href="#"> <img src="images/Kia-Forte-1.png" width="200" height="150"/></a><a data-img="rio-pic" id="rio" href="#"><img id="rio" src="images/Kia-Rio-1.png" width="200" height="150"/></a><a data-img="sorento-pic" id="sorento" href="#"><img src="images/Kia-Sorento-1.png" width="200" height="150"/></a><a data-img="sportage-pic" id="sportage" href="#"><img src="images/Kia-Sportage-1.png" width="200" height="150"/></a></li>
            	<li class="slide"><a data-img="k900-pic" id="k900" href="#"><img  src="images/Kia-K900-1.png" width="200" height="150"/></a><a data-img="optima-pic" id="optima" href="#"><img src="images/Kia-Optima-1.png" width="200" height="150"/></a><a data-img="cadenza-pic" id="cadenza" href="#"><img src="images/Kia-Cadenza-1.png" width="200" height="150"/></a><a data-img="soul-pic" id="soul" href="#"><img src="images/Kia-Soul-1.png" width="200" height="150"/></a></li>
        	</ul>
    	</div>
        
        <hr />
        
        <div class="popup" id="popup" align="center">
        	<div align="right">
        	<a class="button" id="closePopup" align="center" id="closePopup" href="#">&nbsp;X&nbsp;</a>
            </div>
            <b>KIA CLUB</b>
            <div align="center" class="spaceimg"><img src="images/Kia-Logo.png" width="143" height="100"/></div>
            <p>
            Join our mailing list to receive blog posts, exclusive sales information, Kia’s faves and much more!
            </p>
            <p>
    		Name: <input>
    		</p>
            <P>
            Email: <input>
            </P>
            <a class="button" id="join" href="#">&nbsp;&nbsp;Join&nbsp;&nbsp;</a>
        </div>
        
        <div align="center"> <table>
    	<tr>
        <td><div align="left"> <p> According to Kia Motors, the name "Kia" derives from the Sino-Korean words ki ("to come out") and a (which stands for Asia), it is roughly translated as "arise or come up out of Asia" or "rising out of Asia". </p>

			<p>South Korea's oldest car company[citation needed], Kia was founded on December 11, 1944 (when Korea was part of Japan) as a manufacturer of steel tubing and bicycle parts by hand – and has operated as one of the country's Chaebols since then. In 1951, Kia began building complete bicycles. In 1952, Kia changed its name from Kyungsung Precision Industry, and later built Honda-licensed small motorcycles (starting in 1957), Mazda-licensed trucks (1962) and cars (1974). The company opened its first integrated automotive assembly plant in 1973, the Sohari Plant. Kia built the small Brisa range of cars until 1981, when production came to an end after the new military dictator Chun Doo-hwan enforced industry consolidation. This forced Kia to give up passenger cars and focus entirely on light trucks. Prior to the forced 1981 shutdown, Kia rounded out its passenger car lineup with two other foreign models assembled under license: the Fiat 132 and the Peugeot 604.</p>

			<p>Starting in 1986 (when only 26 cars were manufactured, followed by over 95,000 the next year), Kia rejoined the automobile industry in partnership with Ford. Kia produced several Mazda-derived vehicles for both domestic sales in South Korea and for export into other countries. These models included the Kia Pride, based on the Mazda 121, and the Avella, which were sold in North America and Australasia as the Ford Festiva and Ford Aspire.</p>

			<p>In 1992, Kia Motors America was incorporated in the United States. The first Kia-branded vehicles in the United States were sold from four dealerships in Portland, Oregon in February 1994.[citation needed] Since then, Kia methodically expanded one region at a time. Dealers in 1994 sold the Sephia; and a few years later, the United States segment expanded their line with the addition of the Sportage. Over one hundred Kia dealerships existed across thirty states by 1995, selling a record 24,740 automobiles.</p></div> </td>
        <td><div align="center"><img src="images/K900-Gallery-1.jpg" width="200" height="150"/><img src="images/K900-Gallery-2.jpg" width="200" height="150"/><img src="images/K900-Gallery-3.jpg" width="200" height="150"/><img src="images/K900-Gallery-4.jpg" width="200" height="150"/></div></td>
    </tr>
	</table></div>
    
    <hr />
        
	</div>
    
<script src="jquery-1.11.3.min.js"></script>
<script src="Kia Club.js"></script>
</body>
</html>
